js/mandelbrot.js

```diff
@@ -24,22 +24,25 @@ const iterateEquation = (x0, y0, maxiterations) => {
   return iterations;
 }
 
-const scale = (domainStart, domainLength, screenLength, step) =>
-  domainStart + domainLength * ((step - screenLength) / screenLength);
+function scale(domainStart, domainLength, invScreenLength, step) {
+  domainStart + domainLength * (step * invScreenLength - 1);
```

Comment:
you're missing a `return` statement here. Any reason for changing from function expressions to declarations?

Reply:
Yah. Good catch!

Reply:
Fixed